src/software/jetson_nano/ansible/playbooks/robot_auto_test_playbook.yml

```diff
@@ -0,0 +1,66 @@
+---
+
+- name: Robot Auto Test Playbook
+  hosts: THUNDERBOTS_HOSTS
+
+  tasks:
+
+    - debug:
+        msg: "[Robot ID = {{ inventory_hostname }}]"
+      tags: always
+
+    - name: Start Robot Auto Test Process
+      block:
+
+        - name: Stop Services
+          become: true
+          become_method: sudo
+          ansible.builtin.systemd:
+              name: "thunderloop.service"
+              masked: no
+              daemon_reload: yes
+              state: stopped
+          tags: always
+
+        - name: Delete everything on ~/
```

Comment:
Can you update this name to be `Delete robot_auto_test binary under .../thunderbots_binaries/`.